app/templates/core-data/admin-unit/index.hbs

```diff
@@ -40,32 +33,66 @@
                   {{/if}}
                 </:content>
               </Item>
-              <Item><:label>Regio : </:label>
+            {{/if}}
+
+            {{#if @model.isWorshipService}}
+              <Item>
+                <:label>Soort eredienst</:label>
+                <:content
+                >{{@model.adminUnit.recognizedWorshipType.label}}</:content>
+              </Item>
+            {{/if}}
+            {{#if @model.adminUnit.denomination}}
+              <Item>
+                <:label>Strekking</:label>
+                <:content>{{@model.adminUnit.denomination}}</:content>
+              </Item>
+            {{/if}}
+            {{#if
+              (eq @model.adminUnit.constructor.modelName "worship-service")
+            }}
+              <Item>
+                <:label>Grensoverschrijdend</:label>
                 <:content>
-                  {{@model.region}}
-                </:content></Item>
+                  {{@model.adminUnit.crossBorderNominal}}
+                </:content>
+              </Item>
+            {{/if}}
+            {{#if
+              (and @model.adminUnit.scope.locatedWithin @model.isMunicipality)
+            }}
+              <Item>
+                <:label>Regio</:label>
+                <:content>
+                  {{@model.adminUnit.scope.locatedWithin.label}}
+                </:content>
+              </Item>
             {{/if}}
+
+
+
             {{#if @model.isIgs}}
-              {{#if @model.adminUnit.expectedEndDate}}
-                <Item>
-                  <:label>Geplande einddatum</:label>
-                  <:content>
-                    {{date-format @model.adminUnit.expectedEndDate}}
-                    {{#if this.expiredExpectedEndDate}}
-                      <AuPill @skin="error">De geplande einddatum is
-                        overschreden</AuPill>
-                    {{/if}}
-                  </:content>
-                </Item>
-              {{/if}}
-              {{#if @model.adminUnit.purpose}}
-                <Item>
-                  <:label>Doel</:label>
-                  <:content>
-                    {{@model.adminUnit.purpose}}
-                  </:content>
-                </Item>
-              {{/if}}
+              <Item>
+                <:label>Regio</:label>
+                <:content>
+                  {{@model.region}}
+                </:content>
+              </Item>
+              <Item>
+                <:label>Geplande einddatum</:label>
+                <:content>
+                  {{date-format @model.adminUnit.expectedEndDate}}
```

Comment:
`expectedEndDate` and `purpose` are not defined in the frontend model, so if you want to get data you will need to add it there